spec.bs

```diff
@@ -6877,6 +6995,14 @@ result of [=evaluating a bidding script=], or an [=additional bid=] provided by
     was specified, it will have exactly that many [=list/items=]) by the time the bid participates
     in the auction. Must be null if the interest group making this bid has a null
     [=interest group/ad components=] field.
+  : <dfn>selected buyer and seller reporting ID</dfn>
+  :: Null or [=string=]. The selected reporting id from the 
+    [=interest group ad/selectable buyer and seller reporting IDs=] within the
+    [=generated bid/interest group=]. If present, will be passed alongside with
+    [=interest group ad/buyer reporting ID=], and
+    [=interest group ad/buyer and seller reporting ID=] to [=report win=] and with only the

```

Comment:
Should we add [=score ad=] here as well? Maybe ...

```
    If present, this will be:
      - passed alongside [=interest group ad/buyer reporting ID=] and
        [=interest group ad/buyer and seller reporting ID=] to [=report win=] subject to
        [=k-anonymity=] checks.
      - passed alongside [=interest group ad/buyer and seller reporting ID=]
        to [=report result=] subject to [=k-anonymity=] checks.
      - passed alongside [=interest group ad/buyer and seller reporting ID=]
        to [=score ad=].
```